python/Completion.py

```diff
@@ -8,18 +8,31 @@ def get_completions(self, column, partialWord):
         parameters['WantDocumentationForEveryCompletionResult'] = \
             bool(int(vim.eval('g:omnicomplete_fetch_full_documentation')))
 
+        want_snippet = \
+            bool(int(vim.eval('g:omnicomplete_want_snippet')))
+
+        want_method_header = \
+            bool(int(vim.eval('g:omnicomplete_want_method_header')))
+
+        want_return_type = \
+            bool(int(vim.eval('g:omnicomplete_want_return_type')))
```

Comment:
These Vim variable(such as `g:omnicomplete_want_return_type`) have to be initialized at `plugin/OmniSharp.vim`.
Or, this will be an error for the user who doesn't set it.